Cpp_test/lib/ipc-shm-receive.cpp

```diff
@@ -125,12 +126,15 @@ void IPCShmReceive::map_shm()
 	{
 		throw std::runtime_error("ERROR: mmap64() data_ap received.");
 	}
+	this->shm_ptr->is_wait = true;
+	std::cout << this->shm_ptr->is_wait << std::endl;
 }
 
 void IPCShmReceive::open_shm()
 {
 	std::cout << "Waiting for sender..." << std::endl;
-	for (unsigned tries = 0;;)
+	unsigned tries;
```

Comment:
Why is this unsigned? Also for consistency changing the name to time_wait would make sense.

Reply:
This is the old thing when I made shared memory, I just keep the old one. I will change it to time_wait so everything sync